docs/deployed-contracts.md

```diff
@@ -161,6 +177,19 @@
 - wstETH token: [`0xb770Ea0F1762D73c8719B52eF981f7F1D824d9a7`](https://rinkeby.etherscan.io/address/0xb770Ea0F1762D73c8719B52eF981f7F1D824d9a7)
 - Node Operators registry: [`0x776dFe7Ec5D74526Aa65898B7d77FCfdf15ffBe6`](https://rinkeby.etherscan.io/address/0x776dfe7ec5d74526aa65898b7d77fcfdf15ffbe6)
 - DepositContractMock: [`0xd1aC373a6fCAB20476957B14a18178615594Debe`](https://rinkeby.etherscan.io/address/0xd1ac373a6fcab20476957b14a18178615594debe)
+- NodeOperatorsRegistry (Easy Track stub)[`0x7899ef901ed9b331baf7759c15d2e8728e8c2a2c`](https://rinkeby.etherscan.io/address/0x7899ef901ed9b331baf7759c15d2e8728e8c2a2c)
+- Finance (Easy Track stub)|[`0x72cd2b30020fb28de0dfa64d930a183eb732cf46`](https://rinkeby.etherscan.io/address/0x72cd2b30020fb28de0dfa64d930a183eb732cf46)
+
+### Easy Track
+
+- EasyTrack: [`0x661090cB830757B2D0Cd53C49Bf0d17a28992e5d`](https://rinkeby.etherscan.io/address/0x661090cB830757B2D0Cd53C49Bf0d17a28992e5d)
+- EVMScriptExecutor: [`0x2667c7e8731d7314E7c492700Ad440f7C3724184`](https://rinkeby.etherscan.io/address/0x2667c7e8731d7314E7c492700Ad440f7C3724184)
+- IncreaseNodeOperatorStakingLimit: [`0xFE5986E06210aC1eCC1aDCafc0cc7f8D63B3F977`](https://rinkeby.etherscan.io/address/0xFE5986E06210aC1eCC1aDCafc0cc7f8D63B3F977)
+- TopUpLegoProgram: [`0xdA53fF207966b3946facaC52dD22B130D507d276`](https://rinkeby.etherscan.io/address/0xdA53fF207966b3946facaC52dD22B130D507d276)
+- RewardProgramsRegistry: [`0x648C8Be548F43eca4e482C0801Ebccccfb944931`](https://rinkeby.etherscan.io/address/0x648C8Be548F43eca4e482C0801Ebccccfb944931)
+- AddRewardProgram: [`0x3129C041B372eE93a5A8756dc4EC6f154D85Bc9a`](https://rinkeby.etherscan.io/address/0x3129C041B372eE93a5A8756dc4EC6f154D85Bc9a)
+- RemoveRewardProgram: [`0xc21e5e72Ffc223f02fC410aAedE3084a63963932`](https://rinkeby.etherscan.io/address/0xc21e5e72Ffc223f02fC410aAedE3084a63963932)
+- TopUpRewardPrograms: [`0xbb0f594143208eCd04DC7AAe48955902A716F9F7`](https://rinkeby.etherscan.io/address/0xbb0f594143208eCd04DC7AAe48955902A716F9F7)
 
```

Comment:
Maybe also add here addresses of NodeOperatorsRegistry and Finance stubs used in the Rinkeby setup?

Reply:
Hmm, I placed them above originally, but.. ok, just have moved.
Ty!